### ISSUE TITLE
When creating a Case for a Travel Entry, the Disease Variant doesn't get adopted by the new Case

### ISSUE THREAD

**Issue by roxanamlendea** [hzi-braunschweig/SORMAS-Project#8105]
### Bug Description
When a Case is created for a Travel Entry, the disease variant doesn't get adopted by the Case, because the Disease Variant field is missing from the Create form that the user goes through in step 3.

### Steps to Reproduce

1. Open or create a Travel Entry which has 'Covid19' selected as 'Disease' and also has a Disease Variant selected. 
2. Create a Case for the Travel Entry.
3. Save the Create New Case form.

### Actual Behavior
The Disease Variant doesn't get adopted from the Travel Entry to the Case.

### Expected Behavior
The Disease Variant should get adopted from the Travel Entry to the Case.

### Screenshots
![image](https://user-images.githubusercontent.com/89522688/155317794-7d2087fe-f90f-4060-8e6c-77beb2e8de61.png)

![image](https://user-images.githubusercontent.com/89522688/155317848-d786d16c-e575-43a6-af73-738fbef84218.png)

![image](https://user-images.githubusercontent.com/89522688/155317895-7b1c4b62-02e6-41c7-93b6-43a2cafdc60f.png)

### System Details

* Device: Windows 10
* SORMAS version: 1.69.0-SNAPSHOT
* Android version/Browser: Chrome
* Server URL: test-de1.sormas.netzlink.com
* User Role: Admin, NatUser

### Additional Information
The 'Home based quarantine possible' and 'Quarantine' information also don't get adopted when a Case is created for a Travel Entry. I discussed this with @MateStrysewske and he suggested that I should ask you @kwa20 and @JaquM if you think this should be expected or is also a bug.

**Comment by roxanamlendea** [comment]
'Disease Variant Details' field information should also get populated when a Case is created for a Travel Entry. 
 
 The![image](https://user-images.githubusercontent.com/89522688/155982330-5f225153-1965-4ebe-9605-8f4e121c241d.png)
![image](https://user-images.githubusercontent.com/89522688/155982367-06a1aa0e-2291-43e7-a62d-8ee773c55b71.png)

**Comment by roxanamlendea** [comment]
Unfortunately the issue described above (that the 'Disease Variant Details' field doesn't get populated when creating a Case for the Travel Entry) is still reproducible on my local machine.

**Comment by roxanamlendea** [comment]
Verified ticket on local environment using the latest version of Sormas 1.69.0-SNAPSHOT (ef41e6b) from the development branch.

local machine:
![image](https://user-images.githubusercontent.com/89522688/156371333-a909fd4b-8610-42f1-b4b8-815c03aee810.png)

However, I found out that on the testing instances (test-de2, test-de3) the 'Disease Variants' field disappears from the 'Create new case form'. I don't know which configuration could lead to this behaviour.

test-de3.sormas.netzlink.com
![image](https://user-images.githubusercontent.com/89522688/156371592-b34f3c66-4e96-4fe9-b1ae-1fa6f5ba45c6.png)
